src/mlpack/bindings/cli/add_to_po.hpp

```diff
@@ -81,7 +81,7 @@ void AddToPO(const std::string& boostName,
  * used in the CLI function map.
  *
  * @param d Parameter data.
- * @param input Unused void pointer.
+ * @param * (input) Unused void pointer.
```

Comment:
This is how I ended up getting around unused params. I'm not sure if this is desired, but it does seem to work.

Reply:
I found this unhelpful thread: http://doxygen.10944.n7.nabble.com/commented-function-parameters-td3718.html

What do you think about just removing the `@param` documentation for unused parameters?

Reply:
I don't have a particularly strong opinion, I just found this to be the least destructive way around this. Let me know what you think is best.

Reply:
I can see it either way.  With the `@param *`, what actually ends up in the doxygen output?  Is the parameter simply ignored by doxygen?

Reply:
It ends up looking like this:
![Screenshot from 2020-05-13 10-52-40](https://user-images.githubusercontent.com/4951803/81828743-24ecd800-9508-11ea-873a-5ac3046e36ed.png)

Reply:
Hmm, I guess that's okay.  I could go either way on it.  It's nice to list the parameters there, but the `node` and `point` names don't match anything in the given function definition.  So, up to you---probably easier to leave it how it is since you've already gone through everything. :)